### DISCUSSION TITLE
Is "Blockers"tab working as intended?

### DISCUSSION THREAD

**Discussion by hanshenrikbalshoej** [nbrown02/FlowViz#145] [Q&A] [answered]
Hi

I get this picture, when I load data - I know I have blocked tasks and user stories, so I would expect data in the "Current Blocked items" graph. Further "Day lost to being blocked" and "blocker frequency" are empthy

![image](https://github.com/user-attachments/assets/262e95bc-8c01-4ead-a249-968391cc31fe)

Best

Hans Henrik

**Comment by nbrown02** [comment]
Hi Hans,

How are you blocking items?

Tasks won't show as they are excluded...

**Reply by hanshenrikbalshoej** [reply]
We use the status called "Blocked"

![image](https://github.com/user-attachments/assets/bcf96b4a-3044-4a4a-8e98-86574ab62534)

**Reply by nbrown02** [reply, marked as answer]
Hi there,

This isn't accommodated for in the template, please see the top of the [Wiki for the Blockers page overview](https://github.com/nbrown02/FlowViz/wiki/4.-Blockers)

**Comment by hanshenrikbalshoej** [comment]
Thanks for clarifying this - I agree the documentation clearly states this 👍
